### PR TITLE
Update Dependencies

### PR DESCRIPTION
**Pull request by jrjohnson**
Sorry for the large number of changes here. I set out to update just ember-lifeline, but in order to get tests passing I needed to make a bunch of changes all at once and updated ember-cli as well. Hopefully breaking them into several commits will help with reviewing. Let me know if you would rather I squashed this down.

The other thing to be aware of is that I'm not a direct user of this addon. It is a sub-dependency of something else we use. I believe these changes are safe, but someone with actual familiarity with use of this addon should probably test!

### CODE REVIEW

File: addon/components/scroll-content-element.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ export default Ember.Component.extend(DomMixin, {
 
   didInsertElement() {
     this._super(...arguments);
-    this.addEventListener(this.$(), 'scroll', (e) => {
+    this.addEventListener(this.$()[0], 'scroll', (e) => {
```
Comment (alexander-alvarez) [inline addon/components/scroll-content-element.js]: should we prefer `this.element`?

Comment (jrjohnson) [reply addon/components/scroll-content-element.js]: Excellent. I totally forgot about that option. I've made that change now.

**Comment by offirgolan** [comment]
Looks to me as well. You guys want me to issue a patch release with this?

**Comment by jrjohnson** [comment]
@offirgolan a release would be great. Thanks.

**Comment by offirgolan** [comment]
Done! 😸

**Pull request merged**